pdudaemon/drivers/devantech.py

```diff
@@ -153,3 +153,62 @@ def accepts(cls, drivername):
         if drivername == "devantech_eth8020":
             return True
         return False
+
+
+class DevantechDSBase(PDUDriver):
+    port_count = 0
+    connection = None
+
+    def __init__(self, hostname, settings):
+        self.hostname = hostname
+        self.settings = settings
+        self.ip = settings.get("ip", "192.168.0.123")
```

Comment:
It might be better here to have the default IP revert to the hostname, rather than 192.168.0.123

Reply:
Thanks for you review.

I'm sorry, I was not offered an option to simply accept your suggestion.
So I've pushed an identical commit and referenced you.